chia/plotting/util.py

```diff
@@ -233,16 +235,22 @@ def get_filenames(directory: Path, recursive: bool, follow_links: bool) -> list[
         if follow_links and recursive:
             import glob
 
-            files = glob.glob(str(directory / "**" / "*.plot"), recursive=True)
-            for file in files:
+            v1_file_strs = glob.glob(str(directory / "**" / "*.plot"), recursive=True)
+            v2_file_strs = glob.glob(str(directory / "**" / "*.plot2"), recursive=True)
```

Comment:
I think this is quite slow. maybe that's fine for now. We'll probably have 6 months for optimizations between the 3.0 release and activation. An alternative would be to use the glob pattern `*.plot?`

Reply:
although, it will affect everyone. Doubling the plot traversal time

Reply:
ok i understand your concern about using glob X2 (although i wouldn't expect that to be slow just looking at names) 

but why would it double just using *.plot? ? 
i can do that then run once through the strings separating them by type, why would that be slow just iterating through the strings ?

Reply:
i dident expect this to be to much so just doubled the logic, thinking of this we dont really need the sepperation between types so we can just use *.plot{,2}

Reply:
that doesnt work with glob only with pathlib, i think we should keep it and if we want to optimize do that in a new pr